pyproject.toml

```diff
@@ -2,3 +2,18 @@
 line-length = 120
 skip-string-normalization = true
 skip-magic-trailing-comma = true
+
+[tool.isort]
+multi_line_output=3
+include_trailing_comma = true
+line_length = 120
+profile = "black"
+known_first_party = [
```

Comment:
what's doing ?

Reply:
added the isort config as isort recommends pyproject for configuration. But didn't remove it from setup.cfg as i didnt want to break any existing workflow